assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -18,6 +18,9 @@ che.database=${che.home}/storage
 # API service. Browsers initiate REST communications to Che server with this URL
 che.api=http://${CHE_HOST}:${CHE_PORT}/api
 
+# API service internal network url. Back-end services should initiate REST communications to Che server with this URL
+che.api.internal=http://${CHE_HOST}:${CHE_PORT}/api
```

Comment:
is this right to use http, not https?

Reply:
We have https from public routes. But we don't have 'https' for Che services.